.eslintrc.js

```diff
@@ -21,7 +21,8 @@ module.exports = {
     'constructor-super': 'warn',
     'valid-typeof': 'warn',
     'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
-    'prettier/prettier': ['error']
+    'prettier/prettier': ['error'],
+    'react/destructuring-assignment': ['enabled', 'never'],
```

Comment:
Oops, forgot I did this. It's a bit annoying and I don't think we need to change this across the project right now, going to keep it